docs/enterprise-guide/full-app-embedding.md

```diff
@@ -1,85 +1,96 @@
+- **Embedding secret key:** You can ignore this setting, which is only for standalone chart or dashboard embeds.
 
-* **Embedding the entire Metabase app:** Here's where you'll enter the base URLs of the web applications that you want to allow to embed Metabase.
+- **Embedding the entire Metabase app:** Here's where you'll enter the base URLs of the web applications that you want to allow to embed Metabase.
   This value will be used to populate the `Content-Security-Policy` HTTP header's [`frame-ancestors` directive](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors), and should follow the same format.
   For example, `https://*.metabase.com http://my-web-app.example.com:8080/`. Leaving this empty will default to a `frame-ancestors` value of `'none'`.
   If you're a fancy person, you can specify this URL in the environment variable `MB_EMBEDDING_APP_ORIGIN`.
 
 A note to IE11 users: only the first URL will be valid for IE11 due to limitations in the HTTP headers it supports. IE11 does not support the `Content-Security-Policy` header, but does support `X-Frame-Options`, which can only accept a single value.
 
 ### Setting things up in your web app
+
 To give you a picture of what you'll need to do in your app, we've created this [reference app](https://github.com/metabase/sso-examples/tree/master/app-embed-example). If you use React in your application, [this React component](https://github.com/metabase/sso-examples/blob/master/app-embed-example/src/MetabaseAppEmbed.js) may be helpful.
 
 The main elements you'll need to embed Metabase in your app are:
 
-* An SSO authentication endpoint in your application, typically JWT or SAML, and Metabase configured to use them.
-* An `<iframe>` element in your application, with the `src` attribute set to either a URL in Metabase (e.x. `http://metabase.yourcompany.com/dashboard/1`), or directly to the authentication endpoint in your application, with a parameter specifying where to redirect back to (e.x. `http://yourcompany.com/api/auth?redirect=http%3A%2F%2Fmetabase.yourcompany.com%2Fdashboard%2F1`). The latter will avoid an extra redirect if the user has not been authenticated.
-* Optional: JavaScript using [postMessage](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage) in your application for enabling communication to and from the embedded Metabase. Here are the types of `postMessage` messages we currently support:
-
-#### Supported `postMessage` messages *from* embedded Metabase:
+- An SSO authentication endpoint in your application, typically JWT or SAML, and Metabase configured to use them. When using an SSO authentication endpoint on a different domain than Metabase, to avoid problems with `SameSite` cookies, you might need to set the environment variable `MB_SESSION_COOKIE_SAMESITE=None`. The available options are `Lax` (default), `None` or `Strict`.
```

Comment:
The `SameSite` customization isn't aimed at making SSO work across sites, but rather embedding itself. More specifically, it's for the situation where Metabase is hosted on a different domain from the site in which Metabase is embedded.

SSO _should_ be unaffected, since the browser is redirected to the SSO site and the SSO session cookie is therefore set by the first party (the SSO site). Then again, SSO redirection in an `<iframe>` could be a weird edge case.

So this information probably fits better with the next point below, about `<iframe>`s, and reworded to something like this:

> When using an SSO authentication endpoint on a different domain than Metabase, to avoid problems with `SameSite` cookies, you might need to set the environment variable `MB_SESSION_COOKIE_SAMESITE=None`. The available options are `Lax` (default), `None` or `Strict`.